dev-docs/bidders/bluesea.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: bidder
+title: Bluesea
+description: Prebid Bluesea Bidder Adapter
+biddercode: bluesea
+tcfeu_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+media_types: banner, video, native
```

Comment:
Please add the following meta fields

- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- safeframes_ok: true/false

to help others vet your adapter